flink-table/flink-table-common/src/main/java/org/apache/flink/table/connector/source/abilities/SupportsAggregatePushDown.java

```diff
@@ -124,8 +124,6 @@
  *
  * <p>Regardless if this interface is implemented or not, a final aggregation is always applied in a
  * subsequent operation after the source.
- *
- * <p>Note: currently, the {@link SupportsAggregatePushDown} is not supported by planner.
```

Comment:
@iyupeng are we sure that the JavaDocs don't need more updates for this PR?

For example, is this description still correct?

```
 * <p>Note: The local aggregate push down strategy is all or nothing, it can only be pushed down if
 * all aggregate functions are supported.
```

Also maybe we should mention the config option that was added in this PR.

Reply:
Hi @twalthr, the strategy is still `all or nothing` here. `applyAggregates` returns a bool value, meaning whether all local aggregates are accepted by underlying data source or not at all.

The docs could be improved, like mentioning the new option, etc.

I could push a hotfix for docs.

Reply:
That would be great. The more information in the interface description, the better for all implementers.

Reply:
Hi @twalthr I pushed this for doc improvement: https://github.com/apache/flink/pull/17630